server/error-handler.js

```diff
@@ -23,6 +29,10 @@ function handleError(error, req, res, next) {
     } else {
       logger.error(error);
     }
+  } else if (!isRippleError(error)) {
+    // always log stack traces for uncaught exceptions
+    /* eslint-disable no-unused-expressions */
```

Comment:
If it was proper to use short-circuit evaluation for conditionals, then you would never need an "if" statement anywhere,  but we don't do that because it would make code harder to read. Technically any refactoring could be considered a stylistic preference, but that doesn't mean that the preference is arbitrary.